docs/developers/setup-documentation.rst

```diff
@@ -18,13 +18,17 @@ Setting up
 This presumes that you're familiar with the command line, and are using OSX or another UNIX-like system.
```

Comment:
The presumption would be valid if we assumed some base level of knowledge for a "developer" but that word means lots of different things to lots of different people. We do not want to alienate people, and, in fact, want to help educate people who are new at this and help them get started. That's core to our mission. So, I don't ever want to assume that someone is comfortable with the command line or that they already have X or Y installed unless we also provide documentation on how they can get to that point. Let's at least add some links to some of these resources here and/or put them in a separate doc and link to that.